iot-hub/Quickstarts/SimulatedDevice/Program.cs

```diff
@@ -20,23 +21,34 @@ namespace SimulatedDevice
     /// </summary>
     internal class Program
     {
+        //private static Parameters s_parameters;
         private static DeviceClient s_deviceClient;
         private static readonly TransportType s_transportType = TransportType.Mqtt;
 
-        // The device connection string to authenticate the device with your IoT hub.
-        // Using the Azure CLI:
-        // az iot hub device-identity show-connection-string --hub-name {YourIoTHubName} --device-id MyDotnetDevice --output table
-        private static string s_connectionString = "{Your device connection string here}";
-
         private static async Task Main(string[] args)
         {
-            Console.WriteLine("IoT Hub Quickstarts #1 - Simulated device.");
+            Parameters parameters = null;
+            ParserResult<Parameters> result = Parser.Default.ParseArguments<Parameters>(args)
+                .WithParsed(parsedParams => parameters = parsedParams)
+                .WithNotParsed(errors => Environment.Exit(1));
+
+            // The device connection string must be specified in Parameters, or program will exit
+            if (string.IsNullOrWhiteSpace(parameters.DeviceConnectionString))
+            {
+                Console.WriteLine(CommandLine.Text.HelpText.AutoBuild(result, null, null));
+                Environment.Exit(1);
+            }
 
-            // This sample accepts the device connection string as a parameter, if present
-            ValidateConnectionString(args);
+            // The device connection string to authenticate the device with your IoT hub.
+            // Using the Azure CLI:
+            // az iot hub device-identity show-connection-string --hub-name {YourIoTHubName} --device-id MyDotnetDevice --output table
+            //private static string s_connectionString;
+            string connectionString = parameters.DeviceConnectionString;
```

Comment:
Why not pass this directly in as a parameter to the method on 51? Does the alias improve the code?